neptune/alpha/experiment.py

```diff
@@ -95,17 +101,59 @@ def close(self):
             self._bg_job.stop()
             self._op_processor.stop()
 
+    def sync(self, wait: bool = True):
```

Comment:
Let's support python type `datetime` in `experiment.define()`.
And add value type verification there because it's missing.